### PR TITLE
Add issue template with DoD

### PR DESCRIPTION
**Pull request by kasiakepka**
Add issue template with DoD

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+**Reasons**
+
+<!-- Explain why we should add this feature. Provide use cases to illustrate its benefits. -->
+
+**DoD:**
+- [ ] provide documentation
+- [ ] provide unit tests
+- [ ] test on SKR
+- [ ] verify [Get started guides](https://kyma-project.io/docs/kyma/latest/02-get-started/)
```
Comment (strekm) [inline .github/ISSUE_TEMPLATE/feature_request.md:27]: in this case i think it's not needed

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+<!-- Explain why we should add this feature. Provide use cases to illustrate its benefits. -->
+
+**DoD:**
+- [ ] provide documentation
+- [ ] provide unit tests
+- [ ] test on SKR
+- [ ] verify [Get started guides](https://kyma-project.io/docs/kyma/latest/02-get-started/) 
+- [ ] verify resource limits
+- [ ] followup issue
+
```
Comment (strekm) [inline .github/ISSUE_TEMPLATE/feature_request.md:30]: `bump` should be also added in the future, atm we don't know where

**Comment by pbochynski** [comment]
@strekm approve or reject :)

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (kasiakepka) [reply .github/ISSUE_TEMPLATE/feature_request.md:29]: so I am leaving it as it is

**Review by strekm — APPROVED** [review]
/meow

**Review by strekm — APPROVED** [review]

**Pull request merged**